src/floppy/fdc.c

```diff
@@ -1027,9 +1027,9 @@ fdc_write(uint16_t addr, uint8_t val, void *priv)
 						fdc->st0 |= 0x80;
 						drive_num = real_drive(fdc, fdc->drive);
 						/* Three conditions under which the command should fail. */
-						if (!fdd_get_flags(drive_num) || (drive_num >= FDD_NUM) || !motoron[drive_num] || fdd_track0(drive_num)) {
+						if ((drive_num >= FDD_NUM) ||!fdd_get_flags(drive_num) || !motoron[drive_num] || fdd_track0(drive_num)) {
```

Comment:
nitpick: you're missing a space between `||` and `!fdd_get_flags(drive_num)`

Reply:
fixed